### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by emarkees**
Mobile Menu developed using CSS by following the figma file
- Menu Overlay added
- Hamburger added

### CODE REVIEW

**Review by Reem-lab — COMMENTED** [review]
Hi @emarkees, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have the linters set up well since you should add js linters with the HTML and CSS  but I just can see linters for HTML and CSS so kindly add the js as well check this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) ✅✅💯✨🔥

 - [ ] It would be great if you also follow the same order for the given README since your READMe is good but is not follow the same order of [Microverse template](https://github.com/microverseinc/readme-template ) and please don't forget to add the back to top links in the side so please add it 🔥✨💯✅

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by didierganthier** [comment]
Hi Team,

While you put in a lot of hard work in this project, unfortunately, I won't be able to proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project that doesn't have the JavaScript linters. For this project, you must follow these instructions to set them up, [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js). 🔗 
- Also consider fixing your README to match the [template](https://github.com/microverseinc/readme-template). 😄 

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const hamburger = document.querySelector('.hamburger ');
+const navMenu = document.querySelector('.menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
```
Comment (levy002) [inline script.js]: - [x] Kindly make sure when one of the navigation links is clicked, the menu is closed automatically and the user will be redirected to the part of the page according to which was clicked ( FOR EXAMPLE: if the `About` button is clicked, the website will navigate to the `About myself` section).

   - ### YOUR APP BEHAVIOUR

   chrome-capture-2023-2-15

File: style.css [hunk 2]
```diff
@@ -96,20 +94,101 @@ body {
...
+  top: 0;
+  list-style: none;
+  background: #6070ff;
+  -webkit-backdrop-filter: blur(15px);
+  backdrop-filter: blur(10px);
+  background-blend-mode: multiply;
+  opacity: 25;
+  width: 100%;
+  height: 718px;
+}
```
Comment (levy002) [inline style.css]: - [x] KIndly make sure your background is blur as the one the provided Figma design.

- [ ] KIndly make the pop-up menu cover the whole height of the visible screen in order to match the provided Figma design.

 - ### YOUR DESIGN
 
    image

 - ### FIGMA DESIGN
 
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @emarkees   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (anagudelogu) [reply script.js]: - [ ] You did a great job following the previous reviewer's advice on displaying the appropriate section of your page! 🚀 However, when I scroll back to the top, the menu is still open:

[Screencast from 15-03-23 16:33:20.webm](https://user-images.githubusercontent.com/77069011/225449827-55e48475-84a9-479c-b40a-725d6d523153.webm)

Please consider fixing this behavior! According to the requirements: 🤓 🔧 

image

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @emarkees 👋🏼 
I'm Andres.

You've done a great job implementing some of the requested changes for this milestone 🚀, but there are still some which haven't been addressed yet. You're getting closer, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- All changes requested have been implemented  ✅ 
- Cool animation design 👏🏿 
  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**